### PR TITLE
Style alignment

### PR DESCRIPTION
**Pull request by pllaurado**
Style alignment

### CODE REVIEW

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/CaseDetailPage.scala [hunk 1]
```diff
@@ -17,8 +17,8 @@
 package uk.gov.hmrc.tariffclassificationfrontend.views
 
 object CaseDetailPage extends Enumeration {
-  type CaseDetailPage = Value
-  val SUMMARY = Value("summary")
+  type  CaseDetailPage = Value
```
Comment (ed0906) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/CaseDetailPage.scala]: Extra space added here, not that it matters

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/application_details.scala.html [hunk 2]
```diff
@@ -26,166 +26,120 @@
...
-                <div id="app-details-related-reference" class="grid-row">
-                    <div class="column-one-third">
-                        <p>@messages("case.application.related-bti-reference")</p>
-                    </div>
-                    <div class="column-two-thirds">
-                        <p>@btiApp.relatedBTIReference.get</p>
-                    </div>
+                <div class="column-two-thirds">
+                    <p>
+                        @if(btiApp.reissuedBTIReference.isDefined) {
```
Comment (ed0906) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/application_details.scala.html]: You can do this in a `getOrElse`
`btiApp.reissuedBTIReference.getOrElse(messages("answer.no"))`

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/application_details.scala.html [hunk 3]
```diff
@@ -26,166 +26,120 @@
...
-                <div id="app-details-other-info" class="grid-row">
-                    <div class="column-one-third">
-                        <p>@messages("case.application.other-information")</p>
-                    </div>
-                    <div class="column-two-thirds">
-                        <p>@btiApp.otherInformation.get</p>
-                    </div>
+                <div class="column-two-thirds">
+                    <p>
+                    @if(btiApp.relatedBTIReference.isDefined) {
```
Comment (ed0906) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/application_details.scala.html]: Similar to comment above. Can be done with a `getOrElse`

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/application_details.scala.html [hunk 4]
```diff
@@ -26,166 +26,120 @@
...
-                    @attachments_list("attachments", applicationFiles)
-                }
-            }
+            <div id="app-details-legal-proceedings" class="grid-row">
+                <div class="column-one-third">
+                    <p>@messages("case.application.known-legal-proceedings")</p>
+                </div>
+                <div class="column-two-thirds">
+                    <p>
+                    @if(btiApp.knownLegalProceedings.isDefined) {
```
Comment (ed0906) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/application_details.scala.html]: Similar here

File: test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/partials/ApplicationDetailsViewSpec.scala [hunk 5]
```diff
@@ -99,13 +39,11 @@ class ApplicationDetailsViewSpec extends ViewSpec {
...
 
       // Then
-      doc shouldNot containElementWithID("reissued-heading")
-      doc shouldNot containElementWithID("app-details-envisaged-code")
-      doc shouldNot containElementWithID("app-details-confidential-info")
-      doc shouldNot containElementWithID("app-details-related-reference")
-      doc shouldNot containElementWithID("app-details-legal-proceedings")
-      doc shouldNot containElementWithID("app-details-other-info")
-      doc shouldNot containElementWithID("app-details-attachments")
+      doc.getElementById("app-details-reissue-application") should containText("No")
```
Comment (ed0906) [inline test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/partials/ApplicationDetailsViewSpec.scala]: You have access to `messages` here if you dont want to hard code stuff

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/case_trader.scala.html [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+                </div>
+            </div>
+
+            <div class="grid-row">
+                <div class="column-one-third">
+                    <p>@messages("case.application.holder.address")</p>
+                </div>
+                <div class="column-two-thirds">
+                    <p>
+                        @btiHolder.addressLine1 @if(!btiHolder.addressLine1.isEmpty) {<br>}
```
[the remark this answers is not in the document]
Comment (ed0906) [reply app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/case_trader.scala.html]: Also looks like something that should be tested in the TraderViewSpec ;)

[the remark this answers is not in the document]
Comment (pllaurado) [reply app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/case_trader.scala.html]: This fields are not optionals... maybe a function to prettify it?

[the remark this answers is not in the document]
Comment (ed0906) [reply app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/case_trader.scala.html]: Oh theyre strings. Nah leave it its fine

**Pull request merged**